src/features/call/layouts/CallLayout.tsx

```diff
@@ -1,113 +1,87 @@
+import ZUIButton from 'zui/components/ZUIButton';
 
 type Props = {
   callAssId: string;
   children?: ReactNode;
 };
 
 const CallLayout: FC<Props> = ({ callAssId, children }) => {
-  const router = useRouter();
-  const call = useCurrentCall();
   const assignments = useMyCallAssignments();
-
-  const getDetailsPage = (pathname: string) => {
-    if (!pathname) {
-      return false;
-    }
-    const segments = pathname.split('/');
-    if (segments.length === 3 && segments[1] === 'call') {
-      const callId = Number(segments[2]);
-      return !isNaN(callId);
-    }
-    return false;
-  };
-
-  const pathname = usePathname() || '';
-  const isPreparePage = pathname.endsWith('/prepare');
-  const isDetailsPage = getDetailsPage(pathname);
-
   const assignment = assignments.find(
     (assignment) => assignment.id === parseInt(callAssId)
   );
 
-  const { allocateCall } = useAllocateCall(
-    assignment!.organization.id,
-    parseInt(callAssId)
-  );
-  const { deleteCall } = useCallMutations(assignment!.organization.id);
-
   return (
-    <Box
-      sx={{
-        display: 'flex',
-        flexDirection: 'column',
-      }}
-    >
-      <Box
-        sx={{
-          display: 'flex',
-          justifyContent: 'space-between',
-          p: 2,
-        }}
-      >
-        {isDetailsPage && (
-          <Link href="/my/home" passHref>
-            <Button variant="outlined">
-              <Msg id={messageIds.nav.backToHome} />
-            </Button>
-          </Link>
-        )}
-        {isPreparePage && (
-          <Link href="/my/home" passHref>
-            <Button
-              onClick={() => {
-                if (call) {
-                  deleteCall(call.id);
-                }
-              }}
-              variant="outlined"
-            >
-              Stop calling
-            </Button>
-          </Link>
-        )}
-        <Box>
-          {isPreparePage && call && assignment && (
-            <SkipCallDialog
-              assignment={assignment}
-              callId={call?.id}
-              targetName={
-                call?.target.first_name + ' ' + call?.target.last_name
-              }
-            />
-          )}
-          {isDetailsPage && (
-            <Button
-              onClick={() => {
-                router.push(`/call/${callAssId}/prepare`);
-                allocateCall();
-              }}
-              variant="contained"
-            >
-              <Msg id={messageIds.nav.startCalling} />
-            </Button>
-          )}
-          {isPreparePage && <Button variant="contained">Start Call</Button>}
+    <Box>
+      <Box sx={(theme) => ({ backgroundColor: theme.palette.common.white })}>
+        <Box
+          pt={2}
+          sx={{
+            overflow: 'hidden',
+            pl: { sm: 3, xs: 2 },
+            pr: 2,
+            textOverflow: 'ellipsis',
+          }}
+        >
+          <ZUIText noWrap variant="headingMd">
```

Comment:
Same as above: are these two still needed when the noWrap property is on the ZUIText? The intention for the noWrap property is that it should take care of everything needed to truncate the text

Reply:
Fix it too!